packages/core/src/utils/breadcrumbsUtils.ts

```diff
@@ -0,0 +1,20 @@
+import type { Breadcrumb } from '@sentry/types';
+
+/**
+ * Set a Fetch/XHR breadcrumb's log level based on the returned status code
+ * @param breadcrumb
+ */
+export function assignBreadcrumbLogLevel(breadcrumb: Breadcrumb): Breadcrumb {
+  const statusCode = breadcrumb.data?.status_code;
```

Comment:
`FetchBreadcrumbData` and `XhrBreadcrumbData` are sent as the [data](https://github.com/getsentry/sentry-javascript/blob/e0015c51438afa797806de2d7136a5a2bbd4938b/packages/browser/src/integrations/breadcrumbs.ts#L250) property, which optionally contains `status_code`.